apps/wallet/src/features/Send/InputTransaction/TransactionFee/DefaultFee.tsx

```diff
@@ -23,6 +26,12 @@ export const DefaultTransactionFee: FC<Props> = () => {
 
 	const feeString = `${feeAmount ? feeAmount : 0} ${networkToken}`;
 
+	useEffect(() => {
+		if (network && (token || nft) && receiver && Number(amount) > 0) {
+			txActions.updateTransactionFee();
```

Comment:
I think it needed to wrap this function by throttle

Reply:
I assume 300ms was enough. Additionally, I refactor the `txActions.updateTransactionFee` to exclude the transfer amount